edb/server/bootstrap.py

```diff
@@ -1556,12 +1556,17 @@ async def _init_stdlib(
         src_hash=src_hash,
         cache_dir=cache_dir,
     )
-    tpldbdump_package = read_data_cache(
-        tpldbdump_cache,
-        pickled=True,
-        src_hash=src_hash,
-        cache_dir=cache_dir,
-    )
+
+    backend_params = cluster.get_runtime_params()
```

Comment:
Is it ok to execute this piece of code earlier than it was before?

Reply:
Hm..... I think it might actually need to be after `populate_misc_instance_data` to work right?
Let me track it down

Reply:
OK, nevermind, I think it's fine

Reply:
I was thinking of the `edgedb.get_backend_capabilities()` *postgres* function